### PR TITLE
Scav Armor, many bugfixes, map beautification

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

- Restores the Scavenged armor and helmet as a crafting recipe (with new sprites made by me) so that people stop going on Fontaine specific clothing
- Deputy Chief now has access to the bullet fab, as intended
- Removes extant reference to Lonestar by renaming LSS Mineral Water into Spring Water
- Utility closets that previously had nothing but oxygen tanks and masks now have warming up supplies as well, making them actually worth raiding rather than being disappointing fillers
image
- Certain clothing articles now have at least a minimum cold protection when equipped so that it's not the same as if you were stark naked, some values adjusted for specific clothing that's better or worse at covering the cold (like pants and outdoorswear)
- Fixes industrial chem dispenser disappearing
- Fixes makeshift carbine icon disappearing in hands
- New area define for the public firing range
- Beautification of the colony plus a lot of fixes where shutters are involved (PIRS now has a button for theirs, colony lockout more cohesive, etc)

<hr>

## Changelog
:cl:
add: Readds Scavenger armor to crafting menu, replacing craftable Fontaine armor
add: Utility closets that previously had nothing but oxygen tanks and masks now have warming up supplies as well, making them actually worth raiding rather than being disappointing fillers. This includes a self heating cup of coffee akin to an MRE!
add: YEOWCH!
tweak: Moved all fullbody recipes lower on the Clothing tab of the crafting menu, where they belong
tweak: CSO now has the xenobio factory disk mapped rather than part of his locker (it's Provost gear doesn't make sense), moves their medical factory disk to their locker
balance: Clothing parents now have a minimal coverage against cold, to make a difference between being stark naked and not. Covered parts respected as well (pants vs full suits)
fix: Deputy Chief now has training for the bullet fab
fix: Fixes industrial chem dispenser disappearing
fix: Fixes makeshift carbine dual hand icons missing
imageadd: New sprites for scav armor and helmet, new outsider cape and hood sprites (including accessory)
imagedel: Goodbye brown drip for scav armor / outsider
spellcheck: Deletes extant reference to LSS on the mineral water for plants, renames it to Spring Water
map: Beautification of map, includes better colony lockdown blastdoors, buttons for the windows, properly connected lever-to-conveyer belt on the scrap beacon, PIRS gains a button for their shutters, fixed tech shop shutters not closing, adds mapped stops to atmos to prevent upper-to-lower floors (and viceversa) to become cold, and more.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/walllocker.dm [hunk 1]
```diff
@@ -10,15 +10,21 @@
...
 	new /obj/item/clothing/mask/breath(src)
 	new /obj/item/tool/crowbar(src)
+	// Items to heat up in case of extreme cold
+	new /obj/item/clothing/suit/storage/toggle/liberass_jacket(src)
+	if (prob(10)) // Hot ramen, some water and hotsauce, jackpot!
+		new /obj/item/reagent_containers/food/drinks/dry_ramen(src)
+		new /obj/item/reagent_containers/food/drinks/cans/waterbottle(src)
+		new /obj/item/reagent_containers/food/condiment/pack/hotsauce(src)
+	else
+		new /obj/item/reagent_containers/food/drinks/coffee(src)
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/walllocker.dm:27]: wouldnt this be ice coffee from being in the cold rather then anything heating?

Comment (ThePainkiller) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm:27]: > wouldnt this be ice coffee from being in the cold rather then anything heating?

Would you prefer if I coded an MRE-like item that just heated you up if you consume it after crushing it for it to become hot?

Comment (Trilbyspaceclone) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm:27]: Mhm, that would be nice

Comment (ThePainkiller) [reply code/game/objects/structures/crates_lockers/closets/walllocker.dm:27]: Coming soon, just need to localtest to see everything works as intended and I'll push the commit

**Pull request merged**